js/listeners.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+function Listeners() {
+  this.listeners = [];
+}
+
+Listeners.prototype.on = function(elm) {
+  var $elm = $(elm);
+  var args = _.toArray(arguments).slice(1);
+  this.listeners = this._listeners || [];
+  this.listeners.push({$elm: $elm, args: args});
+  $elm.on.apply($elm, args);
+};
+
+Listeners.prototype.off = function() {
```

Comment:
Okay yes, I saw that above here:  https://github.com/18F/fec-style/pull/238/files#diff-55ad661871cd27444f58e59f424ed11bR133 - this makes sense since with the object destroyed, there's no point in having any of the listeners remain for it.

The naming is what threw me off (pun not intended) initially; `clear` sounds a lot better to me in this case because that makes it... clear as to what's happening. :-)